### DISCUSSION TITLE
Google Auth

### DISCUSSION THREAD

**Discussion by Ansh-Garg22** [Note-Vault/Note-Vault#265] [Q&A] [answered]
I've been assigned Issue 241, which involves implementing Google authentication into our project. To proceed with this task effectively, I require certain details, specifically the OAuth 2.0 client ID and client secret.

As we're utilizing Passport.js functionality to support Google authentication, obtaining these credentials is crucial for seamless integration. Could you kindly provide me with the OAuth 2.0 client ID and client secret associated with our project?

**Comment by VaibhavArora314** [comment, marked as answer]
You can use your client I'd and secret for development and use them as environment variables. Specify the names used for the env variables in the .env.example file. 

Also don't push your own credentials, we will set our own credentials as environment on the deployment after merging the pr.

In case you have any queries feel free to ask here.
